test/simulator/moves/belch.js

```diff
@@ -46,12 +46,10 @@ describe('Belch', function () {
 			{species: 'Swalot', ability: 'gluttony', moves: ['toxic']},
 		]);
 		battle.join('p2', 'Guest 2', 1, [{species: 'Registeel', ability: 'prankster', moves: ['rest', 'thunderwave']}]);
-		battle.choose('p2', 'move 2');
-		battle.commitDecisions();
-		battle.choose('p1', 'switch 2'); // For U-Turn
-		battle.choose('p1', 'switch 2');
-		battle.commitDecisions();
-		battle.commitDecisions();
+		battle.makeChoices('move Uturn', 'move Thunderwave');
+		battle.makeChoices('switch 2');//For U-Turn
```

Comment:
I'd prefer `makeChoices('switch 2', '')`.

Make it clearer which side is making the choice.